spring-boot-modules/pom.xml

```diff
@@ -84,6 +84,7 @@
         <module>spring-boot-data-2</module>
         <module>spring-boot-validation</module>
         <module>spring-boot-data-3</module>
+		<module>spring-caching</module>
```

Comment:
We will have to include here spring-caching and spring-caching-2 both

Reply:
Spring-caching-2 is included in the root project of the pom.xml in the sub module style.